en/docs/design/create-api/create-streaming-api/create-a-websub-streaming-api.md

```diff
@@ -0,0 +1,140 @@
+# Create a WebSub API
+
+Follow the instructions below to create a WebSub API using the basic flow.
+
+### Design a WebSub API
+
+1.  Sign in to the WSO2 API Publisher `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher`).
+
+2.  Go to **CREATE API** and Click **Design New Streaming API**.
+
+    [![Design New Streaming API]({{base_path}}/assets/img/learn/design-api/streaming-api/design-new-streaming-api.png)]({{base_path}}/assets/img/learn/design-api/streaming-api/design-new-streaming-api.png)
+
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>The <b>CREATE</b> button will only appear for a user who has the <code>creator</code> role permission.</p>
```

Comment:
Similar to [1].
[1] https://github.com/wso2/docs-apim/pull/3284/files#r589208308